create_marc_record.py

```diff
@@ -0,0 +1,136 @@
+from pymarc import Record, Field, Subfield
+
+
+def create_base_record() -> Record:
+    """Create a base MARC record, to which metadata from an external
+    source will be added."""
+    record = Record()
+
+    # Leader fixed field.
+    # Type (Leader/06) - j (Musical sound recording)
+    record.leader.type_of_record = "j"
+    # BLvl (Leader/07) - m (Monographic/Item)
+    record.leader.bibliographic_level = "m"
+    # Ctrl (Leader/08) - # (No specified type)
+    record.leader.type_of_control = " "
+    # ELvl (Leader/17) - 3 (Abbreviated level)
+    record.leader.encoding_level = "3"
+    # Desc (Leader/18) - i (ISBD punctuation included)
+    record.leader.cataloging_form = "i"
+
+    # 007 - physical description fixed field
+    # 007 ## $a s $b d $d f $e u $f n $g g $h n $i n $m e $n u
+    # translated to fixed field:
+    record.add_field(Field(tag="007", data="sd fungnn|||eu"))
+
+    # 008 - general information fixed field
+    # DtSt (008/06) - s (Single known date/probable date)
+    # Ctry (008/15-17) - xx# (No place, unknown, or undetermined)
+    # Comp (008/18-19) - ##
+    # FMus (008/20) - n (Not applicable)
+    # Part (008/21) - n (Not applicable)
+    # Audn (008/22) - # (Unknown or unspecified)
+    # Form (008/23) - # (None of the following)
+    # AccM (008/24-29) - # (No accompanying matter)
+    # LTxt (008/30-31) - # (Item is a music sound recording)
+    # TrAr (008/33) - n Not arrangement or transposition or not specified
+    # MRec (008/38) - # (Not modified)
+    # Srce (008/39) - d (Other)
+    record.add_field(Field(tag="008", data="YYMMDDs||||    xx ||nn          n ||| d"))
```

Comment:
> 008/22, 008/23, 008/24-29, 008/30-31, 008/32: all blanks
There's one blank missing from the 008 field.  It could be any of these :) but let's call it the 008/32, since that's not listed in the specs.

Per the [008 standard for the Music format](https://www.loc.gov/marc/bibliographic/bd008m.html), the 008/32 is "undefined".  My understanding, per the [previous implementation](https://github.com/UCLALibrary/music-cd-batch/blob/main/old_project/get_music_data.pl#L677), is this should be a blank/space character.

The bib 008 field should be 40 characters long, so I caught this by adding a test for that.